Source/Core/GregorianDate.js

```diff
@@ -25,6 +25,24 @@ function GregorianDate(
   millisecond,
   isLeapSecond
 ) {
+  const minYear = 1;
+  const minMonth = 1;
+  const minDay = 1;
+  const minHour = 0;
+  const minMinute = 0;
+  const minSecond = 0;
+  const minMillisecond = 0;
+
+  if (!year) year = minYear;
+  if (!month) month = minMonth;
+  if (!day) day = minDay;
+  if (!hour) hour = minHour;
+  if (!minute) minute = minMinute;
+  if (!second) second = minSecond;
+  if (!millisecond) millisecond = minMillisecond;
+  validateRange();
```

Comment:
Hi @ggetz I have doubt about the pragmas. If we remove these validations during release then how do we handle the runtime error that may occur?

Reply:
The extra validation step can effect performance, especially if its called every frame. In Cesium, we typically opt to put validation behind the debug pragmas rather than always error check. Errors that occur at runtime can be thrown using `RuntimeError` rather than the `DeveloperError` which is thrown as part of `Check`, but we do not usually consider parameter validation a runtime error.